oeq-ts-rest-api/src/AxiosInstance.ts

```diff
@@ -17,17 +17,19 @@ const catchHandler = (error: AxiosError | Error): never => {
 export const GET = <T>(
   path: string,
   validator?: (data: unknown) => boolean,
-  queryParams?: object
+  queryParams?: object,
+  transformer?: (data: T) => unknown,
 ): Promise<T> =>
   axios
     .get<T>(path, {params: queryParams, paramsSerializer: params => stringify(params)}, )
     .then((response: AxiosResponse<T>) => {
-      if (validator && !validator(response.data)) {
+      const data: any = transformer? transformer(response.data) : response.data;
```

Comment:
I am concerned about if using `any` will break type safety.

Reply:
If I'm interpreting the previous comments for this line correctly right, this still needs to be updated

```suggestion
      const data: unknown = transformer? transformer(response.data) : response.data;
```

Reply:
see https://github.com/PenghaiZhang/openEQUELLA/pull/2 for another approach